src/mod/server.mod/server.c

```diff
@@ -2203,6 +2190,16 @@ if (sasl) {
   my_tcl_strings[0].buf = botname;
   add_tcl_strings(my_tcl_strings);
   add_tcl_ints(my_tcl_ints);
+#ifdef TLS
+#ifndef HAVE_EVP_PKEY_GET1_EC_KEY
+if (sasl) {
```

Comment:
Its better now, but my last comment holds, it should be placed into a tcl write trace function on variable sasl.